tests/token/test_graphql_errors.py

```diff
@@ -0,0 +1,31 @@
+from unittest.mock import AsyncMock
+
+import pytest
+
+from spylib.exceptions import ShopifyGQLError
+
+from ..token_classes import MockHTTPResponse, OfflineToken, test_information
+
+
+@pytest.mark.asyncio
+async def test_store_graphql_happypath(mocker):
+    token = await OfflineToken.load(store_name=test_information.store_name)
+
+    query = """
+    {
+      something {
+        name
+      }
+    }"""
+
+    gql_response = {'data': {}, 'errors': []}
```

Comment:
Why does it raise for this response? Is that the old behaviour?

Reply:
Hey @qw-in , this is a new behavior when the `suppress_errors` is set to `False` to surface the errors mentioned in here https://graphql.org/learn/serving-over-http/#response you brought up in in the issue https://github.com/SatelCreative/spylib/issues/86
>a query might result in some data and some errors,

Reply:
Guess I'm confused - the errors array is empty?

Reply:
oh that.. because it will raise the exception as long as there's the `errors` field, so I thought it didn't matter what's in the array. And I just added in some error message to hopefully eliminate the confusion